src/PolicyInsights/PolicyInsights/help/New-AzPolicyAttestation.md

```diff
@@ -0,0 +1,459 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PolicyInsights.dll-Help.xml
+Module Name: Az.PolicyInsights
+online version:
```

Comment:
Please add online version for each new cmdlet docs. its pattern should be https://learn.microsoft.com/powershell/module/az.policyinsights/{lowerCase(cmdletname)}